src/main/java/org/torproject/onionoo/docs/DocumentStore.java

```diff
@@ -11,7 +11,6 @@
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.PropertyNamingStrategy;
-
```

Comment:
Please avoid unnecessary whitespace changes, if possible. I know, it was more likely the IDE, not you, but maybe you can teach it to not mess with imports unless instructed so.